### ISSUE TITLE
process get_software_versions fails

### ISSUE THREAD

**Issue by t-neumann** [nf-core/rnafusion#65]
Hi,

first question - is there still active development on this?

Second question -  my runs are crashing on the get_software_version process.

```
ERROR ~ Error executing process > 'get_software_versions'

Caused by:
  Process `get_software_versions` terminated with an error exit status (1)

Command executed:

  echo 1.0 > v_pipeline.txt
  echo 19.01.0 > v_nextflow.txt
  fastqc --version > v_fastqc.txt
  multiqc --version > v_multiqc.txt
  cat /groups/zuber/zubarchive/USERS/tobias/.nextflow/assets/nf-core/rnafusion/tools/fusioncatcher/Dockerfile | grep "VERSION" > v_fusioncatcher.txt
  cat /groups/zuber/zubarchive/USERS/tobias/.nextflow/assets/nf-core/rnafusion/tools/fusion-inspector/environment.yml | grep "fusion-inspector" > v_fusion_inspector.txt
  cat /groups/zuber/zubarchive/USERS/tobias/.nextflow/assets/nf-core/rnafusion/tools/star-fusion/environment.yml | grep "star-fusion" > v_star_fusion.txt
  cat /groups/zuber/zubarchive/USERS/tobias/.nextflow/assets/nf-core/rnafusion/tools/ericscript/environment.yml | grep "ericscript" > v_ericscript.txt
  cat /groups/zuber/zubarchive/USERS/tobias/.nextflow/assets/nf-core/rnafusion/tools/pizzly/environment.yml | grep "pizzly" > v_pizzly.txt
  cat /groups/zuber/zubarchive/USERS/tobias/.nextflow/assets/nf-core/rnafusion/tools/squid/environment.yml | grep "squid" > v_squid.txt
  scrape_software_versions.py > software_versions_mqc.yaml

Command exit status:
  1
```

It does not produce all the files it should - and e.g. grepping for `VERSION` in `nf-core/rnafusion/tools/fusioncatcher/Dockerfile` does not deliver any results or grepping for `fusion-inspector` in `nf-core/rnafusion/tools/fusion-inspector/environment.yml` yields actually two lines:

```
name: fusion-inspector
  - fusion-inspector=1.3.1
```

So I was wondering whether this is related to this issue...

**Comment by matq007** [comment]
Hi Tobias,

thank you for reporting the issues. Yes, I am aware of that and it was resolved in the upcoming version `1.0.1`. There should be a release this/next week.

**Comment by t-neumann** [comment]
Hi, is this release happening this week?

**Comment by matq007** [comment]
Hi, sorry for the delay. Yes, we are finishing the last tests. Should be done soon! In the meantime you can join our alack channel where the discussion is hapenning and you can see the status of the testing

**Comment by apeltzer** [comment]
#rnafusion channel here https://nf-core-invite.herokuapp.com/ (but you are already on the slack channel as I can see)

**Comment by t-neumann** [comment]
@apeltzer Thanks yeah I just joined 5 minutes before

**Comment by matq007** [comment]
Fixed in version `1.0.1`
